app/assets/stylesheets/photos.scss

```diff
@@ -0,0 +1,138 @@
+// Place all the styles related to the photos controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+h2 {
+	font-size: 16px;
+}
+
+#post {
+	border: 1px solid #dbc7c7;
+	margin-bottom: 10px;
+	padding-bottom: 5px;
+	background: white;
+}
+
+#postHeader{
+	background-color: white;
+	padding: 0px 10px;
+}
+
+
+#userName {
```

Comment:
Avoid using id selectors<br>Selector `userName` should be written in lowercase with hyphens